object_templates/provider_type.erb

```diff
@@ -3,8 +3,16 @@ require 'puppet/resource_api'
 Puppet::ResourceApi.register_type(
   name: '<%= name %>',
   docs: <<-EOS,
-      This type provides Puppet with the capabilities to manage ...
-    EOS
+@summary Fancy new type
```

Comment:
`@summary a <%= name %> type`

Reply:
Good idea.. I just copy/pasted from the official docs